cf-agent/verify_files.c

```diff
@@ -542,9 +542,14 @@ static PromiseResult VerifyFilePromise(EvalContext *ctx, char *path, const Promi
 
     if (a.haveedit)
     {
-        if (exists)
+        if (exists || StringEqual(a.template_method, "mustache") ||
```

Comment:
> Maybe it would be easier to add the exception(s) here?

This avoids some code, but I don't think this is what you meant?

Reply:
It is.